### PR TITLE
All incomplete

### LINKED ISSUE #28: Mark every Stop as Incomplete
Requested during PVTA meeting on April 30th, 2018:

Make a way to mark every stop as incomplete. @dfaulken and I decided that this should be a rake task.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #28

### CODE REVIEW

**Comment by mboneil10** [comment]
@dfaulken namespaces for the sake of organization/clarity, right? Is there another reason why we use namespaces?

File: lib/tasks/all_incomplete.rake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+namespace :update_all_stops do
+  desc "Mark all stops as incomplete"
+  task :all_incomplete do
```
Comment (werebus) [inline lib/tasks/all_incomplete.rake]: Probably need `:environment` here, right?

Comment (mboneil10) [reply lib/tasks/all_incomplete.rake]: Good call. We're using `uncomplete_all`. Unless anyone has any other ideas.

**Comment by mboneil10** [comment]
I changed the filename although no one had a problem with it... For the sake of future programmers/rake tasks.

**Pull request merged**